Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Adapted from: https://github.com/elasticdog/tiddlywiki-docker/blob/master/5/Dockerfile
+
+ARG NODE_LTS_VERSION=18
+
+FROM node:${NODE_LTS_VERSION}-alpine
+
+ARG TIDDLYWIKI_VERSION=5.2.7
```

Comment:
Forgive my ignorance; can the user override the values that are specified via ARG? I'm thinking it might be useful to give users the option to install the most recent version of TW? Or is it safer to clamp things to a specific version? It may also be useful to be able to specify "master" or another branch name.

Reply:
Yeah a user could do that like so:
```bash
docker build -t tiddlywiki --build-arg TIDDLYWIKI_VERSION=x.y.z .
```

I thought about defaulting to latest but then forgot about it; I'll set that up instead!

Regarding "is it safer", I personally always pin just in case, but I won't force my own paranoia on users! 😂